TA-cyences/metadata/default.meta

```diff
@@ -45,3 +45,7 @@ export = system
 [viewstates]
 access = read : [ * ], write : [ * ]
 export = system
+
+
```

Comment:
Kindly add why we added this in the comment.

# Permission updated so all users can update the notable event status and assignee on Splunk.